.github/workflows/INTEGRATION.yml

```diff
@@ -0,0 +1,39 @@
+name: INTEGRATION
```

Comment:
Any reason why we do have two separate post release builds? I would have expected INTEGRATION and POST_RELEASE to be the same thing, just one matching a less strict tag.

Reply:
That's because of the tags pattern. Anyway, I agree that it might be confusing. Let me try to fix that.

Reply:
https://github.com/bpmn-io/bpmn-js/pull/1441/commits/4b65f512c90b6987ae3bbe92e18d1e11b611f91c solves it in a less confusing manner I believe.